### DISCUSSION TITLE
When building the project, an error is reported: language key type error，version:3.19.0

### DISCUSSION THREAD

**Discussion by LiuCaptain** [amannn/next-intl#1312] [Q&A]
### Description

First of all, I had no problems with local development and local building. The project started and packaged normally. The local packaging is as follows:
![image](https://github.com/user-attachments/assets/3662dbb8-78b6-470e-a435-d5249700335c)

But when I pushed the project to Vercel for packaging, a strange problem occurred, as shown below:
![image](https://github.com/user-attachments/assets/bc1a7bd5-4acb-4141-afef-61d1b6da69ff)
But my multi-language key exists normally:
![image](https://github.com/user-attachments/assets/8031df4f-8914-4583-9c9c-415e5af1d276)
So what is the problem?

### Verifications

- [X] I've verified that the problem I'm experiencing isn't covered in [the docs](https://next-intl-docs.vercel.app/).
- [X] I've searched for similar, existing issues on GitHub and [Stack Overflow](https://stackoverflow.com/search?q=next-intl).
- [X] I've compared my app to [a working example](https://next-intl-docs.vercel.app/examples) to look for differences.

### Mandatory reproduction URL

App Router

### Reproduction description

When packaging and building on vercel, a strange type error occurs immediately

### Expected behaviour

My code is developed and packaged normally locally, and I hope it can be packaged normally on Vercel.

**Comment by amannn** [comment]
I unfortunately also don't see any obvious pointers for the cause of the error.

This seems to be a usage question, I'll move this to a discussion …
